helm/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ This file documents all notable changes to [Gravitee.io API Management 3.x](http
 
 - Add support for multi-server installation
 - Improve redis ratelimit configuration [issues/9726](https://github.com/gravitee-io/issues/issues/9726). Thanks [@gh0stsrc](https://github.com/gh0stsrc)
+- Add support for JWT authentication in HTTP repository (hybrid gateways 'bridge' client)
+- BREAKING CHANGE:
+    - `gateway.management.http.trustall` is now `false` by default. Using a public CA or a well configured truststore should still work
```

Comment:
"Should" or "will"? :)

Reply:
should... prudence is key ;-)
Let say that if their trust store does not contains what is required I can't say it will work...  coz it won't.